cli/cli_app.go

```diff
@@ -52,6 +52,7 @@ const CMD_INIT = "init"
 const CMD_INIT_FROM_MODULE = "init-from-module"
 const CMD_TERRAGRUNT_INFO = "terragrunt-info"
 const CMD_HCLFMT = "hclfmt"
+const CMD_VERSION = "get-terraform-version"
```

Comment:
After taking a look at this, I wonder if this should actually be called `any-action`. Getting the terraform version is the first step right now, but there is no guarantee that this will be the case going forward. And once we shift things around, I don't see too much use in a before hook that wraps the `terraform version` call.

cc @brikis98 : would value a second opinion here.

Reply:
Agreed that `get-terraform-version` is not the right name, as (a) it's specific to our current implementation, which could change and (b) even if it didn't change, there's no reason users should need to know that step is part of the implementation. Not sure `any-action` makes it clear that this would run as one of the very first steps. Perhaps `terragrunt-init` or `terragrunt-run`?

Reply:
I agree terragrunt-init sounds right, however as you said the current implement may change, so terraform version may not be the correct place to latch onto